relay_messages.go

```diff
@@ -184,7 +185,7 @@ func (f *lazyCallReq) Caller() []byte {
 // Service returns the name of the destination service for this callReq.
 func (f *lazyCallReq) Service() []byte {
 	l := f.Payload[_serviceLenIndex]
```

Comment:
actually i wonder if we should just store the service length at L126, maybe save a conversion? likely doesn't impact perf much, but just seems redundant to to it every time we call .Service()

Reply:
Storing would also increase the size of `lazyCallReq` which could have perf impact too. We're just casting the byte to an int, so this shouldn't have any real performance impact.